packages/drawer/src/views/DrawerView.tsx

```diff
@@ -238,7 +238,6 @@ export default function DrawerView({
               renderDrawerContent={renderNavigationView}
               renderSceneContent={renderContent}
               keyboardDismissMode={keyboardDismissMode}
-              drawerPostion={drawerPosition}
```

Comment:
Why did you remove it from here?

Reply:
@satya164 Because it's also a typo and it's duplicated.
[Here (correct)](https://github.com/react-navigation/react-navigation/blob/master/packages/drawer/src/views/DrawerView.tsx#L211) and [here (incorrect and removed)](https://github.com/react-navigation/react-navigation/blob/master/packages/drawer/src/views/DrawerView.tsx#L241)